assets/js/modules/tagmanager/components/common/AccountCreate.js

```diff
@@ -71,6 +71,7 @@ export default function AccountCreate() {
 			<StoreErrorNotices
 				moduleSlug="tagmanager"
 				storeName={ MODULES_TAGMANAGER }
+				shouldDisplayRetry={ false }
```

Comment:
The same point/question applies as for `ErrorNotices`, this component is used in both setup and settings contexts so shouldn't this be a prop and passed `false` only for the setup context?